zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -286,8 +286,6 @@ public void setSelectedProjectType(String selectedProjectType) {
                 && !selectedProjectType.equals("null")) {
             ProjectType projectType = ProjectType.valueOf(selectedProjectType);
             getInstance().setDefaultProjectType(projectType);
-        } else {
-            getInstance().setDefaultProjectType(null);
```

Comment:
When I test it after the change, upon opening up project home, one of the request (I think it's one of the ajax request but hard to verify) keep call this method and set the project type to null. This method is meant to be called only if user changes project type and there is no way they can select null. So I remove it.